Lib/threading.py

```diff
@@ -899,6 +900,8 @@ class is implemented.
         self._args = args
         self._kwargs = kwargs
         if daemon is not None:
+            if daemon and not _daemon_threads_allowed():
+                raise RuntimeError('daemon threads are disabled in this interpreter')
```

Comment:
perhaps word the error message here and below as "daemon threads are disabled in this (sub)interpreter" to provide a better hint as to why.

Reply:
fixed